src/MembershipsSDKAdapter.test.js

```diff
@@ -32,10 +32,12 @@ describe('Memberships SDK Adapter', () => {
                 {
                   id: 'id',
                   muted: false,
+                  sharing: false,
                 },
                 {
                   id: 'mutedPerson',
                   muted: true,
+                  sharing: true,
```

Comment:
It's funny that the muted person is sharing 😂

Reply:
Yes, it's funny, but it could be a real case :))